INV4/pallet-ipl/src/benchmarking.rs

```diff
@@ -0,0 +1,23 @@
+//! Benchmarks for IPL Pallet
+#![cfg(feature = "runtime-benchmarks")]
+
+pub use super::*;
+use frame_benchmarking::{benchmarks, impl_benchmark_test_suite, whitelisted_caller};
+use frame_system::RawOrigin;
+use primitives::OneOrPercent;
+
+benchmarks! {
+  set_permission {
+      let s in 0 .. 100;
+      let caller: T::AccountId = whitelisted_caller();
+      let sub_asset: T::IplId = Default::default();
+  }: _(RawOrigin::Signed(caller), T::IplId::from(s), sub_asset, Default::default(), true)
+
+  set_asset_weight {
+      let s in 0 .. 100;
+      let caller: T::AccountId = whitelisted_caller();
+      let sub_asset: T::IplId = Default::default();
+  }: _(RawOrigin::Signed(caller), T::IplId::from(s), sub_asset, OneOrPercent::One)
```

Comment:
The IPL hasnt been created prior to running these, same issue as burning without minting first

Reply:
Check